.github/workflows/ci.yaml

```diff
@@ -8,21 +8,47 @@ on:
     branches:
       - master
 
+env:
+  MAVEN_OPTS: '-Xms2048m -Xmx2048m'
```

Comment:
Why add this?

Reply:
More memory means faster compilation. I may have used too few here, actually. Then, high min heap means less growing, which is faster, too.